charts/seed-controlplane/charts/etcd/values.yaml

```diff
@@ -4,6 +4,10 @@ replicas: 1
 images:
   etcd: image-repository:image-tag
```

Comment:
This won't be needed anymore in etcd chart and should be moved to etcd-duird chart as per the conversation at https://github.com/gardener/gardener/pull/1762/files#r361963465

Reply:
changed the logic to add the images